src/pybind/mgr/dashboard/frontend/src/app/core/navigation/navigation/navigation.component.html

```diff
@@ -104,6 +104,12 @@
             <a i18n
                routerLink="/alerts">Alerts</a>
           </li>
+          <li routerLinkActive="active"
+              class="tc_submenuitem tc_submenuitem_prometheus"
+              *ngIf="prometheusConfigured && permissions.prometheus.read">
+            <a i18n
+               routerLink="/silence">Silences</a>
```

Comment:
Since both `Alerts` and `Silences` are related to `Prometheus` and you are really configuring alerts/silences for `Prometheus` and not the cluster itself, how about having a single menu called `Prometheus`?

It would be the same as the `Block iSCSI` or `Block Images` where a single menu has multiple related tabs.

Reply:
@LenzGr had the idea to put it into cluster, as my first proposal was a menu called `Prometheus`. But maybe now is the right time to move it in it's own menu as we now have 2 Prometheus related menu entries.